src/components/ec-main-container/ec-main-container.vue

```diff
@@ -3,42 +3,49 @@
     class="ec-main-container"
     data-test="ec-main-container"
   >
+
     <div
-      v-if="title || hasSlot('cta')"
-      class="tw-grid"
+      v-if="hasSlot('breadcrumbs')"
```

Comment:
if there is no `breadcrumbs` slot then we won't show `cta` slot, I guess it's not wanted behaviour.

Reply:
I see we have another v-if below